packages/stellar-axelar-std/src/traits.rs

```diff
@@ -1,3 +1,28 @@
+extern crate alloc;
```

Comment:
Feature gate this under a new `alloc` feature in `Cargo.toml`, so it's not included by default. ITS cargo.toml can explicitly enable `alloc` feature on stellar-axelar-std